src/audio/Audio.js

```diff
@@ -95,7 +95,7 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 		var source = this.context.createBufferSource();
 
 		source.buffer = this.buffer;
-		source.detune.value = this.detune;
+		this.setDetune( this.detune );
```

Comment:
I think there is a problem if you set `detune` like this. Since `isPlaying` is still false at this line, the value is actually not set to the `AudioBufferSourceNode`.

Reply:
Merging this for now so we no longer break in iOS 12.